src/sea.h

```diff
@@ -1,8 +1,7 @@
-#pragma once
-#include <stdio.h>
+#ifndef _SEA_H
```

Comment:
Not of a fan of `#pragma` ? ^_^ We should probably lay out what OS/compiler combos (or C std) we want to support here.